console/react/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
This file was created by a tool that initializes the react app. It can used by phones/tablets to put an icon on their screen to open the console. 
I've updated the text.